RR.App.Tests/Controllers/HRIS/PropertyAccessControllerUnitTests.cs

```diff
@@ -5,22 +5,27 @@
 using Moq;
 using RR.App.Controllers.HRIS;
 using RR.Tests.Data;
-using RR.Tests.Data.Models.HRIS;
 using Xunit;
+using HRIS.Services.Services;
+using Microsoft.AspNetCore.Http;
+using HRIS.Services.Session;
+using RR.App.Tests.Helper;
 
 namespace RR.App.Tests.Controllers.HRIS;
 
 public class PropertyAccessControllerUnitTests
 {
-
     private readonly Mock<IPropertyAccessService> _propertyAccessMockService;
     private readonly Mock<IEmployeeService> _employeeMockService;
     private readonly PropertyAccessController _propertyAccessController;
+    private readonly Mock<AuthorizeIdentity> _authorizeIdentityMock;
 
-    public PropertyAccessControllerUnitTests() {
+    public PropertyAccessControllerUnitTests()
+    {
+        _authorizeIdentityMock = new Mock<AuthorizeIdentity>(MockBehavior.Strict, null, null);
         _propertyAccessMockService = new Mock<IPropertyAccessService>();
         _employeeMockService = new Mock<IEmployeeService>();
-        _propertyAccessController = new PropertyAccessController(new AuthorizeIdentityMock(), _propertyAccessMockService.Object, _employeeMockService.Object);
+        _propertyAccessController = new PropertyAccessController(new AuthorizeIdentityMock(1), _propertyAccessMockService.Object, _employeeMockService.Object);
```

Comment:
Either use `AuthorizeIdentityMock` or `Mock<AuthorizeIdentity>`  in your tests. I see you using both. 

Just for consistency.